pkg/networkmanager/interfaces.js

```diff
@@ -1722,6 +1722,19 @@ PageNetworking.prototype = {
             self.rx_plot.resize();
             self.tx_plot.resize();
         });
+
+        const desc = _("Network bonding is a method to combine or aggregate network interfaces to provide a logical interface with higher throughput or redundancy.");
+        const lm = _("Learn more");
+        const url = "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/managing-networking-in-the-web-console_system-management-using-the-rhel-8-web-console#bond-modes_configuring-network-bonds-using-the-web-console";
+        const popover_content = desc + '<a href="' + url + '" target="_blank" rel="noopener noreferrer"> <i class="fa fa-external-link fa-xs"></i>' + lm + '</a>';
```

Comment:
Seems a bit odd to not leave a space between desc and the `<a>`, but then put the space inside of it (before the `<i`)